docs/.eleventy.js

```diff
@@ -134,6 +134,35 @@ module.exports = function(eleventyConfig) {
     return output;
   });
 
+  // Copy button shortcode
+  eleventyConfig.addLiquidShortcode("copybutton", function(name) {
+    var checkmarkIcon = Icons["Checkmark"];
+    var copyIcon = Icons["Copy"];
+    var tooltipId = "tooltip-" + (name || Math.floor(Math.random() * 1000));
+
+    var output = '';
```

Comment:
Is there something stopping you from using a template string (` `` `) for this block?

Reply:
Nothing's stopping me from using a template string. Originally, I lifted the implementation from other similar functions within this file out of ~~laziness~~ consistency, but using template strings here does make it much more readable/maintainable. Commit incoming.